### PR TITLE
feat: update floating menu

### LINKED ISSUE #673: ⚡ Feature | Update the Plus Menu to latest version
Because we are adding Twitter embeds and other commands like audio and video, we need to update the Plus menu.

This update include some small UI changes:

1 - Change the Subparagraph font style on DESKTOP and MOBILE from 14px **to 13px**
Capture d’écran 2022-09-14 à 16 45 00

2 - Use "subparagraph" font style for the Plus menu instead of "Subheading" if not set correctly
Capture d’écran 2022-09-14 à 16 46 49

3 - Change border radius of the whole menu from 4px **to 8px**

4 - Update padding left and right from 12px **to 16px** and padding top and bottom from 4px **to 8px**
Capture d’écran 2022-09-14 à 16 49 10
Capture d’écran 2022-09-14 à 16 49 59

5 - set spacing between elements **to 0px** instead of 12px
Capture d’écran 2022-09-14 à 16 51 14

6 - Add "Plain text" command on top of the menu
Capture d’écran 2022-09-14 à 16 51 59

7 - Make sure the order of the commands is correctly set
Capture d’écran 2022-09-14 à 16 53 24

8 - Add categories "Basics" and "Embeds" as shown on the component (12px padding on top of the separator, 8 px bottom padding): https://www.figma.com/file/uPVMeBq0bERyViiRMXzLN8/Sigle-design-system?node-id=3462%3A9512
Capture d’écran 2022-09-14 à 16 55 02

9 - Add the Twitter embed command
Capture d’écran 2022-09-14 à 16 58 13

**ICONS:**

Style=Bullet list
Style=Code Block
Style=Divider
Style=H1
Style=H2
Style=H3
Style=Image
Style=Number list
Style=Quote
Style=Text
Style=Twitter embed

### PR DESCRIPTION
**Pull request by gregogun**
In this pr, the following changes have been made to the floating menu:
- Font size of floating menu items updated
- Menu has now been split into sections with labels
- Twitter item added

Closes #673

### CODE REVIEW

File: sigle/src/modules/editor/extensions/SlashCommand/CommandList.tsx [hunk 1]
```diff
@@ -133,6 +157,26 @@ export const CommandList = forwardRef<CommandListRef, CommandListProps>(
             )}
           </CommandsListItem>
         ))}
+        <CommandsListSeparator />
+        <CommandsListLabel>Embeds</CommandsListLabel>
```
Comment (pradel) [inline sigle/src/modules/editor/extensions/SlashCommand/CommandList.tsx:161]: If we were to merge that pr now, users would see the Twitter embed, I think this change should go in the twitter pr or under a flag :)

File: sigle/src/icons/TwitterLight.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { IconProps } from '@radix-ui/react-icons/dist/types';
+
+export const TwitterLight = ({ width = 28, height = 28 }: IconProps) => (
```
Comment (pradel) [inline sigle/src/icons/TwitterLight.tsx:3]: Here we should have a consistent default heigh / width across all the icons

**Review by pradel — COMMENTED** [review]
Love the new style of this menu 🚀 

There is smth weird going on with the border-radius here, the color is darker than the rest.

Screenshot 2022-09-22 at 10 20 13

File: sigle/src/modules/editor/extensions/SlashCommand/CommandList.tsx (hunk 1 above)
Comment (gregogun) [reply sigle/src/modules/editor/extensions/SlashCommand/CommandList.tsx:161]: ah yes sure I'll remove and add it to the twitter pr 👍

File: .changeset/cuddly-tables-kick.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+'@sigle/app': minor
+---
+
+Updates floating menu items font size.
+Adds sections with labels.
+Adds a Twitter embed item.
```
Comment (pradel) [inline .changeset/cuddly-tables-kick.md:7]: could you please update this part before we merge ? :)

**Review by pradel — APPROVED** [review]

**Pull request merged**